### PR TITLE
docs: translate and correct documents

### PR DESCRIPTION
**Pull request by misitebao**
- [x] Updated English translation
- [x] Delete unnecessary files
- [x] Add Chinese new document
- [x] Fix crowdin configuration
- [x] Keep the new document filename consistent with the old filename style
- [x] Add documentation issue template

### CODE REVIEW

**Comment by leaanthony** [comment]
Thanks for opening this! 👍 Are those style changes necessary? They seem to work as is?

**Comment by misitebao** [comment]
The same is true for static resources. If it is not fixed, in "start" mode, other languages than the default language will not display pictures normally. The specific reasons can be found [here](https://docusaurus.io/docs/static-assets#in-jsx)

**Comment by misitebao** [comment]
@leaanthony 
Preparations have been made, after which the document translation will be taken over by "Crowdin".

File: crowdin.yml [hunk 1]
```diff
@@ -1,9 +1,33 @@
+project_id: 527906
+api_token_env: CROWDIN_PERSONAL_TOKEN
```
Comment (misitebao) [inline crowdin.yml:2]: This is mainly used in conjunction with the vscode plugin. For details, please refer to: https://marketplace.visualstudio.com/items?itemName=Crowdin.vscode-crowdin

**Pull request merged**